server/logging/logger.go

```diff
@@ -59,6 +65,7 @@ func NewLoggerFromLevel(lvl LogLevel) (*logger, error) {
 
 	return &logger{
 		LoggerFacade: ctxLogger,
+		Closer:       &closer{},
```

Comment:
The closer should be the underlying structuredLogger, so that we can flush out the logs with the sync fxn